tests/crashes/131451.rs

```diff
@@ -0,0 +1,9 @@
+//@ known-bug: #131451
+//@ needs-rustc-debug-assertions
+//@ compile-flags: -Zmir-opt-level=5 -Zvalidate-mir
```

Comment:
Please don't do this. Enable the MIR optimizations that are required instead.

Reply:
is there a way to get the set of passes that had been run so far?

Reply:
I tried ` -Zvalidate-mir --crate-type lib  -Zmir-enable-passes=+CheckAlignment,+LowerSliceLenCalls,+BeforeInline,+Inline,+RemoveStorageMarkers,+RemoveZsts,+RemoveUnneededDrops,+UnreachableEnumBranching,+UnreachablePropagation,+SimplifyCfg::AfterUnreachableEnumBranching,+ReferencePropagation,+ScalarReplacementOfAggregates,+MatchBranchSimplification,+MultipleReturnTerminators,+InstSimplify::AfterSimplifyCfg,+SimplifyLocals::BeforeConstProp,+DeadStoreElimination::Initial,+GVN,+SimplifyLocals::AfterGVN,+DataflowConstProp,+SingleUseConsts` but to no avail  :/

Reply:
I moved this one into a separate pr https://github.com/rust-lang/rust/pull/133104